src/content/docs/journey-performance/slo-tutorial.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: 'Create front-end performance benchmarks with service levels'
+metaDescription: 'Learn how to create and define performance benchmarks with servic levels.'
+---
+
+import sonicSliUiOverview from 'images/sonic_screenshot-full_sli-ui-overview.png'
+
+## Objectives
+
+By the end of this tutorial, you will:
+
+* Understand service level indicators (SLIs) and service level obectives (SLOs).
+* Have created and defined SLIs and SLOs for your front-end experience.
+* Have alerts set up to warn of any SLI or SLO breaches so you can react before they impact your users.
+
+## Intro 
+
+Service levels describe services provided to users within a given period of time, in measurable terms. Service level objectives (SLOs) are the goals set for the availability expected out of a system. Service level indicators (SLIs) are the key measurements and metrics to determine the availability of a system. 
+
+Defining and managing your service levels following the steps below provides the ability to: 
+
+* Ease future setup: Automatically establish a baseline of performance and reliability for any service with a one-click setup and recommendations and customizations provided in a simple, guided flow.
+* Define reliability across teams: Avoid arduous alignment processes with SLO and SLI recommendations that help you determine service boundaries. Set reliability benchmarks automatically based on recent performance metrics in any entity.
+* Iterate and improve: With full-stack context and automation through open-source infrastructure-as-code tools like Terraform, teams have insight into how specific nodes or services impact system reliability and can quickly take control over their performance. Custom views for both service owners and business leaders drive operational efficiency and lead to better reporting, alerting, and incident management processes.
+* Standardize reliability: Cross-organizational teams have a unified, transparent view of service reliability, and can better comply with customer-facing SLAs. SLO compliance metrics and error budgets give organizations a way to report on reliability and implement changes across applications, infrastructure, and teams in a cohesive fashion.
+
+Let's get started:
```

Comment:
I would expand upon this (I like get started statements) just a bit. Maybe add in a sentence or something to drive the reader to action/follow along with the procedure.